opm/input/eclipse/Schedule/Action/ActionParser.cpp

```diff
@@ -259,7 +260,7 @@ ASTNode Parser::parse_and()
     auto curr = this->current();
     if (curr.type == TokenType::op_and) {
         ASTNode and_node(TokenType::op_and);
-        and_node.add_child(left);
+        and_node.add_child(std::move(left));
```

Comment:
I was a bit worried by the `return left` at the end of the function, but I see that it is ok since we return from within this if. It would be more readable if you move the `return left` to an `else` branch though.

I see this got merged while I wrote this... Not a big problem in any case.

Reply:
> It would be more readable if you move the `return left` to an `else` branch though.

I'll take this into account for follow-up work.  I have some other changes planned in this area.